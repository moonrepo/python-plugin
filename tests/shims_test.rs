use proto_pdk_test_utils::*;

#[cfg(not(windows))]
generate_global_shims_test!("python-test", ["pip"]);
