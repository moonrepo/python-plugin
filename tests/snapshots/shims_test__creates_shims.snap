---
source: tests/shims_test.rs
expression: "std::fs::read_to_string(sandbox.path().join(\".proto/shims/registry.json\")).unwrap()"
---
{
  "pip": {
    "parent": "python-test",
    "before_args": [
      "-m",
      "pip"
    ]
  },
  "python-test": {},
  "python-test3": {
    "alt_bin": true,
    "parent": "python-test"
  }
}
